org.armagetronad.ArmagetronAdvanced.json

```diff
@@ -50,6 +54,16 @@
                         "autoreconf -vfi"
                     ]
                 }
+            ],
+            "post-install": [
+                "sed -i /app/share/metainfo/*.appdata.xml -e 's,org.armagetronad.org.armagetronad,org.armagetronad,' -e 's,.desktop</id>,</id>,' -e 's,<!-- <launchable,<launchable,' -e 's,</launchable> -->,</launchable>,'",
```

Comment:
Protip: The releases section should also be possible to be added/updated by sed, but I am too dumb (and lazy) to do it properly. :-)

Reply:
I... tend to just pick out a unique substring of the line I want to replace, then replace the whole line with that on with the line I want. It's my hammer. Or rather, that's what I should do, because if I don't, I create mess like org.armagetronad.org.armagetronad.